packages/editor-ui/src/components/ExpressionEditModal.vue

```diff
@@ -155,7 +155,6 @@ async function onDrop(expression: string, event: MouseEvent) {
 					:mapping-enabled="!isReadOnly"
 					:connection-type="NodeConnectionType.Main"
 					pane-type="input"
-					context="modal"
```

Comment:
@elsmr this was introduced [here](https://github.com/n8n-io/n8n/pull/11645) but i believe is no longer needed

Reply:
Correct this was only to have unique id's for the checkboxes that powered collapse/expand